From 104311443d40797b61daf18b6e73b98dd67f4fc3 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Thu, 6 Sep 2018 10:38:00 +0800
Subject: [PATCH] arm: dts: rk3128h-box: Add regulator-early-min-microvolt
 property to the vdd_arm node

Need to keep the vdd_arm voltage set by the bootloader until all consumer
complete the initialization of the frequency voltage constraint.

The previous kernel was using the property regulator-init-microvolt.
The commit 8726e76f589c ("Revert "regulator: of: Use regulator-init-microvolt as early minimum"")
change it to regulator-early-min-microvolt property.

Change-Id: If86994dd3f6a845efbde6cbadfbd73e3572a544b
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/boot/dts/rk3128h-box.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3128h-box.dtsi b/arch/arm/boot/dts/rk3128h-box.dtsi
index cac444ca027a..44a077469fb2 100644
--- a/arch/arm/boot/dts/rk3128h-box.dtsi
+++ b/arch/arm/boot/dts/rk3128h-box.dtsi
@@ -208,6 +208,7 @@
 		compatible = "pwm-regulator";
 		rockchip,pwm_id = <1>;
 		regulator-init-microvolt = <1200000>;
+		regulator-early-min-microvolt = <1200000>;
 		pwms = <&pwm1 0 5000 1>;
 		regulator-name = "vdd_arm";
 		regulator-min-microvolt = <950000>;
-- 
2.35.3

